modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-connector-reference.adoc

```diff
@@ -0,0 +1,526 @@
+| User Properties a| Object |  The custom user properties that should be set to this Anypoint MQ Message |  |
+| Message Id a| String |  A custom ID for the outgoing Anypoint MQ Message. This value must be a proper UID for the message to avoid delivery problems. If absent, the broker will create one UUID for each message. |  |
+| Send Content-Type a| Boolean |  Set to true if the body type should be sent as a Anypoint MQ Message property |  true |
+| Target Variable a| String |  Name of a variable to store the operation's output. |  |
+| Target Value a| String |  An expression to evaluate against the operation's output and store the expression outcome in the target variable |  #[payload] |
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors |  |
+|===
+
+==== Output
+[%autowidth.spread]
+|===
+|Type |Binary
+| Attributes Type a| <<AnypointMQMessagePublishAttributes>>
+|===
+
+==== For Configurations
+* <<config>>
+
+==== Throws
+* ANYPOINT-MQ:PUBLISHING
+* ANYPOINT-MQ:CONNECTIVITY
+* ANYPOINT-MQ:ILLEGAL_BODY
+* ANYPOINT-MQ:DESTINATION_NOT_FOUND
+* ANYPOINT-MQ:RETRY_EXHAUSTED
+
+
+== Sources
+
+[[subscriber]]
+=== Subscriber
+`<anypoint-mq:subscriber>`
+
+
+Anypoint MQ Subscriber Message Source retrieves messages from the given destination name.
+
+
+==== Parameters
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use. | | x
+| Queue a| String |  Name of the Queue from which messages will be retrieved. |  | x
+| Subscriber Type a| One of:
+
+* <<polling>>
+* <<prefetch>> |  The strategy to use when subscribing for messages from the service |  |
+| Acknowledgment Mode a| Enumeration, one of:
+
+** IMMEDIATE
+** AUTO
+** MANUAL |  Acknowledgment mode to use for the messages retrieved. |  AUTO |
+| Acknowledgment Timeout a| Number |  The default acknowledgmentTimeout for all the subscribers, unless it is overridden. Duration that a message is held by a broker waiting for an Acknowledgment or Not Acknowledgment. After that duration elapses, the message is again available to any subscriber. |  0 |
+| Acknowledgment Timeout Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  Time unit to use in the acknowledgmentTimeout configuration |  MILLISECONDS |
+| Circuit Breaker a| <<circuitBreaker>> |  |  |
+| Output Mime Type a| String |  The mime type of the operation output. |  |
+| Output Encoding a| String |  The encoding of the operation output. |  |
+| Primary Node Only a| Boolean |  Whether this source should only be executed on the primary node when running in Cluster |  |
```

Comment:
should be executed only on the primary node   (placement of "only")

Cluster > a cluster